### PR TITLE
Merge the upstream v0.9.0 tag

### PR DESCRIPTION
**Pull request by timflannagan**
```
$ git remote -v
origin	https://github.com/timflannagan/operator-framework-rukpak (fetch)
origin	https://github.com/timflannagan/operator-framework-rukpak (push)
rukpak	https://github.com/operator-framework/rukpak (fetch)
rukpak	https://github.com/operator-framework/rukpak (push)
upstream	https://github.com/openshift/operator-framework-rukpak (fetch)
upstream	https://github.com/openshift/operator-framework-rukpak (push)
```

and

```
$ git merge --theirs v0.9.0 --allow-unrelated-histories
```

### CODE REVIEW

**Comment by tylerslaton** [comment]
Hard to parse all of the changes but it seems as simple as merging with the v0.9.0 tag as you mentioned. That + updating the .gitignore and the golangci-lint version look good to me.

/lgtm

**Comment by tylerslaton** [comment]
Looks like `envtest` is failing to start in the unit tests

/hold

**Comment by timflannagan** [comment]
Alright should be good now after testing locally.

/hold cancel

**Comment by timflannagan** [comment]
This is largely a no-op while we bootstrap the o/platform-operators repository to be introduced into the payload.

/label px-approved
/label qe-approved
/label docs-approved
